src/main/kotlin/io/snyk/plugin/services/SnykApplicationSettingsStateService.kt

```diff
@@ -37,7 +37,8 @@ class SnykApplicationSettingsStateService : PersistentStateComponent<SnykApplica
     var advisorEnable: Boolean = true
     var snykCodeSecurityIssuesScanEnable: Boolean = true
     var snykCodeQualityIssuesScanEnable: Boolean = false
-    var iacScanEnabled: Boolean = false
+    var iacScanEnabled: Boolean = true
```

Comment:
Are we already enabling IaC with the next release?

Reply:
Enablement for the end users hidden by the feature flag.
Here we just change default enablement for IaC product according to Georgi's decision.